hw/dv/tools/dvsim/common_modes.hjson

```diff
@@ -52,19 +52,19 @@
     // in two ways. One of them is setting the pre-processor macro `BUILD_SEED` to the seed value,
     // which is done below. The SystemVerilog testbench sources can use the `BUILD_SEED` macro
     // value to set some design constants (such as parameters) upon instantiation. The `BUILD_SEED`,
-    // if not set externally (by passing the --build-seed switch) is set to 1 in 
-    // `hw/dv/sv/dv_utils/dv_macros.svh`. The other way is by passing the {seed} value to utility 
-    // scripts that generate packages that contain randomized constants. These utility scripts can 
-    // be invoked as a `pre_build_cmd`, wrapped within the `build_seed` sim mode in the DUT 
+    // if not set externally (by passing the --build-seed switch) is set to 1 in
+    // `hw/dv/sv/dv_utils/dv_macros.svh`. The other way is by passing the {seed} value to utility
```

Comment:
```suggestion
    // is set to 1 in `hw/dv/sv/dv_utils/dv_macros.svh` if not set externally by passing the --build-seed switch).
    //. The other way is by passing the {seed} value to utility
```

Also, remove the trailing `,' in the line above.

By the way, otp also uses a "build" image.

Reply:
Amusingly, the change actually landed in April(!) with f4c2bb9 and the hunk will go away when I rebase in a sec.